packages/angular-output-target/src/plugin.ts

```diff
@@ -42,11 +42,5 @@ export function normalizeOutputTarget(config: Config, outputTarget: OutputTarget
     results.directivesArrayFile = normalizePath(path.join(config.rootDir, outputTarget.directivesArrayFile));
   }
 
-  if (outputTarget.includeSingleComponentAngularModules !== undefined) {
```

Comment:
Should this still be considered experimental?

Reply:
I don't know at this point. It felt strange that we are adding a new output type that isn't experimental, but keeping the scam generation as experimental. 

While Angular is pushing hard on the ecosystem to lean towards standalone, existing applications and large companies just aren't going to move in that direction quickly or at all. The need for maintaining the feature will exist for a few years at least, which seems like a long time to keep it in an experimental stage. 

I'd be happy to revert my changes at this part of work (keeping `scam` as experimental) and then open a design doc for discussion to move it out of experimental. Feels the most appropriate with our process 👍

Reply:
I agree with the points you brought up regarding SCAMs. I think reverting the change + a design document makes sense here since we can get some cross-team discussions going.

Reply:
Addressed here: https://github.com/ionic-team/stencil-ds-output-targets/pull/365/commits/a2ff7a7c1059bf55026687a5a4559f05c9870d73

Also add an error for developers currently using `includeSingleComponentAngularModules` option to migrate to `outputType` (seemed like a minimal DX touch to help reduce confusion).